src/prefect/projects/steps/core.py

```diff
@@ -1,21 +1,62 @@
 """
 Core primitives for managing Prefect projects.
 """
+import subprocess
+import sys
+from typing import Optional
+
 from prefect.utilities.importtools import import_object
 
+RESERVED_KEYWORDS = {"requires"}
+
+
+def _get_function_for_step(fully_qualified_name: str, requires: Optional[str] = None):
+    try:
+        step_func = import_object(fully_qualified_name)
+        return step_func
+    except ImportError:
+        if requires:
+            print(
```

Comment:
not necessary for merging into the branch, but i suspect we'll want to elevate this to a logger